aclabs/aclabs/settings.py

```diff
@@ -79,8 +79,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'aclabs2020',
+        'USER': 'admin',
+        'PASSWORD': 'rootroot',
```

Comment:
i'd rather not have the password in clear like now. or i can make the repo private, but then it will only be visible to contributors (meaning us)
could you use a `import os; os.environ.get('DB_PASSWORD', ''),` instead? or should i make the repo private?

Reply:
No, we certainly aren't going to leave the password in clear, in fact the host and port might change as well as I might recreate the db, this was just for test purposes to see if we can successfully connect the backend to the db.